docs/UserGuide.md

```diff
@@ -147,11 +147,16 @@ Examples:
 
 Shows the details of a particular member in the IG.
 
-Format: `details NAME`
+Format: `details NAME` or `details INDEX`
 
+* The argument interprets as an `INDEX` if it is a positive integer, `NAME` otherwise.
+* Displays the details of the person at the specified `INDEX`.
+* The index refers to the index number shown in the displayed person list.
+* The index **must be a positive integer** 1, 2, 3, …​
 * The NAME entered must match the member’s name exactly (case-sensitive).
 
 Examples:
+* `list` followed by `details 3` displays details of the 3rd person in the displayed person list.
 * `details Xiao Ming` will output:
```

Comment:
Actually I realise that the output currently doesn't have the labels next to the detail (e.g. Name:, Phone:, ...)
Do we want to add that in or just change the UG?
![Capture](https://user-images.githubusercontent.com/31680580/137083226-812f37ad-54a4-4275-b73b-47168885a003.PNG)